metsrw/mets.py

```diff
@@ -492,20 +522,18 @@ def _analyze_fptr(fptr_elem, tree, entry_type):
 
     @staticmethod
     def _add_dmdsecs_to_fs_entry(elem, fs_entry, tree):
-        dmdids = elem.get("DMDID")
-        if dmdids:
```

Comment:
I might be missing something and I guess it's an edge case, but what's the reason for removing this check? In the new implementation if there's no DMDIDs the other side effects (reordering and mapping setting) will run.

Reply:
It was just to remove indentation in this block and it will loop over an empty set in all cases, which shouldn't be a major performance hit.